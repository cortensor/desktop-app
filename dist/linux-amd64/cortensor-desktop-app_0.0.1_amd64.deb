version https://git-lfs.github.com/spec/v1
oid sha256:3aca410682875beea02064ef91a0a35a0c766583f055836e9409d9dd6dacfc4d
size 224137362
