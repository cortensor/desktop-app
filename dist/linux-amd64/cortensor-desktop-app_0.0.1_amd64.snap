version https://git-lfs.github.com/spec/v1
oid sha256:1709753d8b81719435009b1e694134a75091512877a194314126734d7180f341
size 296796160
